courseraresearchexports/commands/jobs.py

```diff
@@ -0,0 +1,329 @@
+        'id',
+        help='Export request ID')
+
+    parser_download.add_argument(
+        '--dest',
+        default='.',
+        help='Destination folder')
+
+    parser_clickstream_links = jobs_subparsers.add_parser(
+        'clickstream_download_links',
+        help='Get download links for completed eventing exports.')
+    parser_clickstream_links.set_defaults(func=get_clickstream_links)
+
+    create_scope_subparser(parser_clickstream_links)
+
+    parser_clickstream_links.add_argument(
+        '--interval',
+        nargs=2,
+        metavar=('START', 'END'),
+        help='Interval of exported clickstream data, inclusive. '
+        '(i.e. 2016-08-01 2016-08-04).')
+
+    return parser_jobs
+
+
+def create_scope_subparser(parser):
+    scope_subparser = parser.add_mutually_exclusive_group(
+        required=True)
+    scope_subparser.add_argument(
+        '--course_id',
+        help='Export rows corresponding to learners within a course according '
+        'to the unique id assigned by Coursera.')
+    scope_subparser.add_argument(
+        '--course_slug',
+        help='Export rows corresponding to learners within a course according '
+        'to the unique name of your course defined as the part after '
+        '/learn in the course url. (e.g. machine-learning for '
+        'https://www.coursera.org/learn/machine-learning).')
+    scope_subparser.add_argument(
+        '--partner_id',
+        help='Export rows corresponding to learners within a partner.')
+    scope_subparser.add_argument(
+        '--partner_short_name',
+        help='Export rows corresponding to learners within a partner.')
+    scope_subparser.add_argument(
+        '--group_id',
+        help='Export rows corresponding to learners without a group.')
+
+
+def create_request_parser(subparsers):
+    parser_request = subparsers.add_parser(
+        'request',
+        help=request.__doc__,
+        description=request.__doc__)
+    parser_request.set_defaults(func=request)
+    request_subparsers = parser_request.add_subparsers()
+
+    # common arguments between schema and eventing exports
+    request_args_parser = argparse.ArgumentParser(add_help=False)
+
+    create_scope_subparser(request_args_parser)
+
+    request_args_parser.add_argument(
+        '--statement_of_purpose',
+        required=True,
+        help='Please let us know how you plan to use the '
+        'data, what types of research questions you\'re asking, who will '
+        'be working with the data primarily, and with whom you plan to '
+        'share it.')
+
+    # tables subcommand
+    parser_tables = request_subparsers.add_parser(
+        'tables',
+        help='Create a data export request for specified tables.',
+        description='Create a data export request for specified tables.',
+        parents=[request_args_parser])
+
+    parser_tables.add_argument(
+        '--export_type',
+        choices=exports.constants.EXPORT_TYPE_TABLES,
```

Comment:
Doesn't seem like it... I separated into `request_clickstream` and `request_tables` to avoid this floating argument and set the `export_type` directly.